indico/migrations/versions/20231107_0956_492c6d801a4a_add_google_wallet_integration.py

```diff
@@ -0,0 +1,33 @@
+"""Add Google Wallet integration
+
+Revision ID: 492c6d801a4a
+Revises: 13d8ba15a83b
+Create Date: 2023-11-07 09:56:13.034271
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '492c6d801a4a'
+down_revision = '31b699664893'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('forms', sa.Column('ticket_google_wallet_enabled', sa.Boolean(), nullable=False,
+                                     server_default='false'), schema='event_registration')
```

Comment:
server_default should be removed (`op.alter_column()` with `server_default=None`) after creating the column since it would result in a mismatch between the DB and what the models specify (the `default` there is python-side)